src/sagemaker_training/mpi.py

```diff
@@ -95,18 +95,20 @@ def run(
 
         if wait:
             logger.info("Waiting for MPI process to finish.")
-            _wait_orted_process_to_finish()
+            self._wait_master_to_finish()
             logger.info("Orted process exited")
```

Comment:
Since we are not checking orted any more, I think the above log can be removed to avoid confusion.

Reply:
will this lead to unexpected hangs?

Reply:
@roywei In what sense? This is happening on worker nodes and we are moving to `sleep` rather than this. https://psutil.readthedocs.io/en/latest/#psutil.wait_procs

Reply:
sorry I missed the `_wait_master_to_finish` part,  +1 to Indu's concern here: https://github.com/aws/sagemaker-training-toolkit/pull/145#pullrequestreview-1084524955